src/contracts/Staking.sol

```diff
@@ -137,25 +139,54 @@ contract Staking is Ownable {
     }
 
     /**
-        @notice override whether or not deposits are blocked
+        @notice override whether or not staking is paused
+        @dev used to pause staking in case some attack vector becomes present
         @param _shouldPause bool
-        **/
+     */
     function shouldPauseStaking(bool _shouldPause) public onlyOwner {
         pauseStaking = _shouldPause;
     }
 
     /**
-        @notice override whether or not withdraws are blocked
+        @notice override whether or not unstaking is paused
+        @dev used to pause unstaking in case some attack vector becomes present
         @param _shouldPause bool
-        **/
+     */
     function shouldPauseUnstaking(bool _shouldPause) external onlyOwner {
         pauseUnstaking = _shouldPause;
     }
 
+    /**
+        @notice set epoch length
+        @dev epoch's determine how long until a rebase can occur
+        @param length uint
+     */
+    function setEpochLength(uint256 length) external onlyOwner {
```

Comment:
do we have test for modifying this mid cycle and making sure nothing breaks with out accounting / warm ups / etc?

Reply:
So there are no tests, but the epoch length will not affect the current epoch in anyway, only the next one.  The warmup is based on epochs, so the length is agnostic from that.  Still, I will mess around with tests surrounding this thought because it seems like there should be a test surrounding this logic.  This is a good one, thanks!

Reply:
Will add in a new PR